File: programmers/etc/level3/숫자_게임.py
def solution(A, B):
    answer = 0
    A.sort(); B.sort()
    
    j = 0
    for i in range(len(A)):
        if A[j] < B[i]:
            answer += 1
            j += 1

    return answer


A, B = [5,1,3,7], [2,2,6,8]
print(solution(A, B)) # 3

A, B = [2,2,2,2], [1,1,1,1]
print(solution(A, B)) # 0

File: Java/programmers/etc/level3/숫자게임.java
package CodeTest.Java.programmers.etc.level3;

import java.util.*;

public class 숫자게임 {
    public int solution(int[] A, int[] B) {
        int answer = 0;
        Arrays.sort(A); Arrays.sort(B);
        
        for (int i=0, j=0; i<A.length; i++) {
            if (A[j] < B[i]) {
                answer++;
                j++;
            }
        }
        
        return answer;
    }
}
